diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/FileSize.kt

```diff
@@ -27,31 +31,18 @@ class FileSize(private val configRules: List<RulesConfig>) : Rule("file-size") {
         emitWarn = emit
         isFixMode = autoCorrect
         if (node.elementType == ElementType.FILE) {
-            fileName = node.getFileName()
-            val configuration = FileSizeConfiguration(
-                this.configRules.getRuleConfig(FILE_IS_TOO_LONG)?.configuration ?: mapOf()
-            )
-            val ignoreFolders = configuration.ignoreFolders
-
-            val realFilePath = calculateFilePath(fileName)
-
-            if (!realFilePath.contains(SRC_PATH)) {
+            val filePathParts = node.getFilePath().splitPathToDirs()
+            if (SRC_PATH !in filePathParts) {
                 log.error("$SRC_PATH directory is not found in file path")
             } else {
-                if (ignoreFolders.none { realFilePath.containsAll(it.splitPathToDirs()) }) {
+                val ignoreFolders = configuration.ignoreFolders
```

Comment:
Maybe it's better here without a variable?

Reply:
Yes, makes sense